src/rules/trailingCommaRule.ts

```diff
@@ -86,21 +187,21 @@ class TrailingCommaWalker extends Lint.AbstractWalker<Options> {
                 case ts.SyntaxKind.NamedImports:
                 case ts.SyntaxKind.NamedExports:
                     this.checkList((node as ts.ArrayLiteralExpression | ts.BindingPattern | ts.NamedImportsOrExports).elements,
-                                   node.end);
+                                   node.end, node);
```

Comment:
add the check if the node should be handled in this switch statement instead of passing `node` and adding another big switch statement